l10n_it_fatturapa_out/models/account.py

```diff
@@ -81,6 +89,9 @@ def preventive_checks(self):
                     )
                 )
 
+            if not invoice.partner_id.city:
```

Comment:
Sto guardando le specifiche in https://www.agenziaentrate.gov.it/portale/web/guest/specifiche-tecniche-versione-1.7 perché il link nel README a https://www.fatturapa.gov.it/export/fatturazione/it/normativa/f-2.htm risponde 404.
Da quel che ho capito (penso che qui tu stia risolvendo https://github.com/OCA/l10n-italy/pull/2563#issuecomment-1023962861), il nodo obbligatorio è `Comune`, ed è obbligatorio ogni volta che compare nel XML.

Il template però prende il campo `city` non solo del partner della fattura: https://github.com/OCA/l10n-italy/blob/f114377637b06c15c03903bdc2358aaf8d05d0cc/l10n_it_fatturapa_out/data/invoice_it_template.xml#L464-L466
ma anche del partner della company:
https://github.com/OCA/l10n-italy/blob/f114377637b06c15c03903bdc2358aaf8d05d0cc/l10n_it_fatturapa_out/data/invoice_it_template.xml#L331-L333
e altri, tipo:
https://github.com/OCA/l10n-italy/blob/f114377637b06c15c03903bdc2358aaf8d05d0cc/l10n_it_fatturapa_out/data/invoice_it_template.xml#L338-L343

quindi se vogliamo fare controlli preventivi per non far fallire la generazione del XML, ci sarebbe da controllare il campo `city` di tutti i partner coinvolti, che ne pensi?

Reply:
Fatto.